mqt/bench/utils/utils.py

```diff
@@ -355,14 +356,7 @@ def postprocess_ocr_qasm_files():
 
 
 def create_zip_file():
-    zf = ZipFile("MQTBench_all.zip", "w")
-    for dirname, subdirs, files in os.walk(get_qasm_output_path()):
-        for filename in files:
-            if filename.endswith(".qasm"):
-                zf.write(
-                    os.path.join(dirname, filename),
-                    compress_type=ZIP_LZMA,
-                    compresslevel=3,
-                    arcname=filename,
-                )
-    zf.close()
+    return subprocess.call(
+        "zip -rj ./mqt/benchviewer/static/files/MQTBench_all.zip ./mqt/benchviewer/static/files/qasm_output/",
```

Comment:
This does not use the same compression algorithm as the Python source before, right?
In the Python code, the LZMA algorithm is used.

Reply:
That is true. But actually the Python code never worked and this is also the same command which was used in the current MQT Bench setup. Do you think this is a problem?

Reply:
If it worked fine until now, then it should not be a problem 👍🏼